browser.js

```diff
@@ -395,6 +395,71 @@ function write (buffer, string, offset, length, encoding) {
   return codecFor(encoding).write(buffer, string, offset, length)
 }
 
+function writeDoubleLE (buffer, value, offset) {
+  offset ??= 0
```

Comment:
Logical nullish assignment is only supported from Node.js 15 and onwards so let's do `if (offset === undefined) offset = 0` as in the existing functions instead.